assets/src/application/ui/display/icons/svgs/index.ts

```diff
@@ -43,6 +47,6 @@ export { ReactComponent as WarningTriangle } from './warningTriangle.svg';
 
 // Enhanced icons
 
-export const InfoCircleOutlined = withEnhance(InfoCircleOutlinedSVG);
+export const InfoCircleOutlined = pipe(withGeneratedClassName, withSize)(InfoCircleOutlinedSVG);
 
-export const GlobalOutlined = withEnhance(GlobalOutlinedSVG);
+export const GlobalOutlined = pipe(withGeneratedClassName, withSize)(GlobalOutlinedSVG);
```

Comment:
the order of HOCs that are applied matters, tried the other way and it wasn't working as expected